devito/passes/clusters/implicit.py

```diff
@@ -170,7 +172,7 @@ def callback(self, clusters, prefix):
                 continue
 
             # Get the dynamic thickness mapper for the given MultiSubDomain
-            mapper, dims = lower_msd(d.msd, c)
+            mapper, dims = lower_msd({i.dim for i in ispace}, c)
```

Comment:
this is just `ispace.itdims`

Reply:
It'll get forgotten if it isn't done here, so will just do it